### ISSUE TITLE
Fails to run on Ubuntu 18.04 LTS

### ISSUE THREAD

**Issue by adamplumb** [lapce/lapce#134]
I downloaded the binary from the homepage today and attempted to run.  I'm running Ubuntu 18.04.  Yep it's pretty old, though still under active support from Canonical.  Feel free to close this if Ubuntu 18 isn't a target for you but I figured I would file a ticket just in case.

```
$ ./lapce
./lapce: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by ./lapce)
./lapce: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.29' not found (required by ./lapce)
./lapce: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by ./lapce)
```

Here's my `uname` output:
```
$ uname -a
Linux aplumb-Blade 5.4.0-104-generic #118~18.04.1-Ubuntu SMP Thu Mar 3 13:53:15 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

Looks like I'm running glibc 2.27

**Comment by leonqli** [comment]
Same here.

**Comment by dypa** [comment]
you can try build binary in docker container

create container `docker pull ubuntu:18.04 && docker run -it ubuntu:18.04 `

in docker container run commands

> apt update
> apt install -y curl git
> curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
> source $HOME/.cargo/env
> git clone https://github.com/lapce/lapce.git /opt
> apt install -y cmake pkg-config libfreetype6-dev libfontconfig1-dev libxcb-xfixes0-dev libxkbcommon-dev python3
> cd /opt/lapce
> cargo build --release

then copy binary to host system
`docker cp <containerId>:/opt/lapce/target/release/lapce .`

**Comment by dypa** [comment]
fixed in 912768066e5bc942c7bc9b08146882b355a18923

**Comment by panekj** [comment]
Lapce is built on ubuntu 18.04 for now so this can be closed
